fragments/yrewrite_metainfo/head.php

```diff
@@ -1,15 +1,12 @@
 <meta charset="utf-8" />
 <meta name="viewport" content="width=device-width, initial-scale=1" />
 <?php
-
-// YRewrite + URL
-use Url\Seo;
-
-$seo = new Seo();
-if (!is_array($seo->getTags())) {
-    echo $seo->getTags();
+if (rex_addon::get('url') && rex_addon::get('url')->isAvailable()) {
```

Comment:
`rex_addon::get('url')` liefert immer ein Objekt (`rex_addon` oder `rex_null_addon`), somit wird der erste Teil niemals `falsy` sein.

Somit:
```suggestion
if (rex_addon::get('url')->isAvailable()) {
```

Reply:
Mensch weißt du nicht dass ich jetzt wieder all meinen Code überall so anpasse.... :-)
Thx!